build.gradle

```diff
@@ -18,7 +18,7 @@ java.toolchain.languageVersion = JavaLanguageVersion.of(8)
 group = "micdoodle8.mods.galacticraft"
 archivesBaseName = "Galacticraft"
 
-version = '4.0.4-SNAPSHOT'
+version = '4.0.4-tts-SNAPSHOT'
```

Comment:
All the changes here are out of scope for PR, nor do they fix any issues it may have, If you must change a build file in a project locally, ensure it is not commited.